docs/src/storage.md

```diff
@@ -467,16 +467,43 @@ To use a pre-provisioned volume in CloudNativePG:
     cluster.) Make sure you check for any pods stuck in `Pending` after you deploy
     the cluster. If the condition persists, investigate why it's happening.
 
-## Block storage considerations (Ceph/ Longhorn)
-
-Most block storage solutions in Kubernetes recommend having multiple replicas
-of a volume to improve resiliency. This works well for workloads that don't
-have resiliency built into the application. However, CloudNativePG has this
-resiliency built directly into the Postgres `Cluster` through the number of
-instances and the persistent volumes that are attached to them. 
-
-In these cases, it makes sense to define the storage class used by the Postgres
-clusters as one replica. By having additional replicas defined in the storage
-solution (like Longhorn and Ceph), you might incur what's known as write
-amplification, unnecessarily increasing disk I/O and space used.
-
+## Block storage considerations (Ceph/Longhorn)
+
+Most block storage solutions in Kubernetes, such as Longhorn and Ceph,
+recommend having multiple replicas of a volume to enhance resiliency. This
+approach works well for workloads that lack built-in resiliency.
+
+However, CloudNativePG integrates this resiliency directly into the Postgres
+`Cluster` through the number of instances and the persistent volumes attached
+to them, as explained in ["Synchronizing the state"](architecture.md#synchronizing-the-state).
+
+As a result, defining additional replicas at the storage level can lead to
+write amplification, unnecessarily increasing disk I/O and space usage.
+
+In such cases, consider reducing the number of replicas at the block storage
```

Comment:
I don't love "in such cases" here. Do we mean "cases where the application handles resiliency above the block level"? Or do we mean, "when using CloudNativePG"?
```suggestion
For CloudNativePG usage, consider reducing the number of replicas at the block storage
```

Reply:
I suggest you directly make the change @jsilvela so that we can have this doc in the next release. Thanks.